packages/webdriver/src/utils.js

```diff
@@ -415,3 +415,25 @@ export function getEnvironmentVars({ isW3C, isMobile, isIOS, isAndroid, isChrome
         isSeleniumStandalone: { value: isSeleniumStandalone }
     }
 }
+
+/**
+ * Decorate the params object with host updates based on the presence of
+ * directConnect capabilities in the new session response. Note that this
+ * mutates the object.
+ * @param  {Object} params    post-new-session params used to build driver
+ */
+export function setupDirectConnect(params) {
+    log.info(JSON.stringify(params))
```

Comment:
Do we need this info, if we have another one a couple lines down?

Reply:
nope, this was for debugging. will remove